opslevel/resource_opslevel_user.go

```diff
@@ -92,86 +92,83 @@ func (r *UserResource) Schema(ctx context.Context, req resource.SchemaRequest, r
 				},
 			},
 			"skip_welcome_email": schema.BoolAttribute{
-				DeprecationMessage: "The skip_welcome_email attribute is deprecated and only kept for backward compatibility.",
-				Description:        "Don't send an email welcoming the user to OpsLevel. Applies during creation only, this value cannot be read or updated.",
-				Optional:           true,
+				Description: "Don't send an email welcoming the user to OpsLevel.",
+				Optional:    true,
```

Comment:
can we make the default "true" - as thats generally what people are going to want.